src/components/slider/ImageSlider.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { FaArrowAltCircleRight, FaArrowAltCircleLeft } from 'react-icons/fa';
+import PropTypes from 'prop-types';
+import SliderData from './SliderData';
+
+const ImageSlider = ({ slides }) => {
+  const [current, setCurrent] = useState(0);
+  const { length } = slides;
+
+  const nextSlide = () => {
+    setCurrent(current === length - 1 ? 0 : current + 1);
+  };
+
+  const prevSlide = () => {
+    setCurrent(current === 0 ? length - 1 : current - 1);
+  };
+
+  if (!Array.isArray(slides) || slides.length <= 0) {
+    return null;
+  }
```

Comment:
- `slides` should be a string according to your propTypes, why do you test whether it's an array ?
- You could remove the whole condition and do something like:
```
return slides.length &&     <section className="slider">[...]</section
```